autograph.yaml

```diff
@@ -833,6 +907,8 @@ authorizations:
           - appkey2
           - appkey3
           - normankey
+          - normandy
```

Comment:
This should be either `normankey` or `normandy`, but it should be associated to the `normandev` authorization